usecases/replica/coordinator.go

```diff
@@ -159,5 +154,45 @@ func (c *coordinator[T]) Fetch(ctx context.Context, cl ConsistencyLevel, op read
 		close(replyCh)
 	}()
 
-	return replyCh, level, nil
+	return replyCh, state, nil
+}
+
+func (c *coordinator[T]) Fetch2(ctx context.Context, cl ConsistencyLevel, op readOp2[T]) (<-chan simpleResult[T], rState, error) {
```

Comment:
Did you intend to leave this method name as `Fetch2`?

Reply:
Fetch is there just for backward comp
It will be removed and replaced by Fetch2 once Exists() has been implemented